measures/measures-schema.yaml

```diff
@@ -287,18 +287,17 @@ definitions:
     anyOf:
       - required: ['procedureCodes']
       - required: ['diagnosisCodes']
-    required: [submissionMethods]
 
   performanceOption:
     type: object
     properties:
       submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
       optionType:
-        enum: [performanceMet, performanceNotMet, performanceExclusion]
+        enum: [performanceMet, performanceNotMet, performanceExclusion, performanceException]
```

Comment:
Same comment here around `eligiblePopulationExclusion/Exception`

Reply:
@philipdickinson you're PR can update these to be `performanceMet, performanceNotMet, eligiblePopulationExclusion, eligiblePopulationException` and then the other code can map correctly to these new names.